### DISCUSSION TITLE
See progress from debug/run commands

### DISCUSSION THREAD

**Discussion by augustocdias** [mrcjkb/rustaceanvim#418] [Q&A] [answered]
When I run one of the debugables or runnables options it runs in the background. Is there a way to see the compilation/running progress? Can the output be shown in a split or float window?

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave:

You can set `vim.g.rustaceanvim.tools.test_executor`, see `:h RustaceanToolsOpts`.
The following executor aliases display the output in a split: `"termopen" | "quickfix" | "toggleterm" | "vimux"`
Note that `toggleterm` and `vimux` require the respective backends.

**Reply by mrcjkb** [reply]
P.S. I may change the default back to `termopen` (which is the default for neovim 0.9) if it turns out people prefer it.
